app/assets/scss/application.scss

```diff
@@ -23,60 +23,33 @@ $path: "/static/images/";
 
 // Digital Marketplace Front-end toolkit styles
 @import "toolkit/_breadcrumb.scss";
-@import "toolkit/_browse-list.scss";
 @import "toolkit/_buttons.scss";
-@import "toolkit/_contact-details.scss";
 @import "toolkit/_document.scss";
 @import "toolkit/_grids.scss";
 @import "toolkit/_instruction-list.scss";
-@import "toolkit/_link-button.scss";
 @import "toolkit/_notification-banners.scss";
 @import "toolkit/_page-headings.scss";
 @import "toolkit/_phase-banner.scss";
 @import "toolkit/_proposition-header.scss";
 @import "toolkit/_secondary-action-link.scss";
-@import "toolkit/_service-id.scss";
-@import "toolkit/_temporary-message.scss";
 @import "toolkit/forms/_dates.scss";
 @import "toolkit/forms/_hint.scss";
-@import "toolkit/forms/_keyword-search.scss";
-@import "toolkit/forms/_option-select.scss";
 @import "toolkit/forms/_questions.scss";
 @import "toolkit/forms/_selection-buttons.scss";
 @import "toolkit/forms/_summary.scss";
 @import "toolkit/forms/_textboxes.scss";
 @import "toolkit/forms/_validation.scss";
```

Comment:
Near the top of the file we are importing `forms/word-counter` and `forms/list-entry` both from the toolkit. I think they could both be moved into this section with the rest of the toolkit imports (I can't think of a reason why the order would matter but worth just double checking).

Reply:
Are they being imported from the same place? Why don't they have ```toolkit/``` in the beggining?

Reply:
Maybe they are being imported from the other location (Sam mentioned there were two locations we were pulling in the toolkit, one of those being bower). I would assume it's safe to import them using the way we are doing the rest of our toolkit imports. However you can also spin the apps up and find a question with a word limit to check the word limit styling is working successfully.

Reply:
Cool, I've changed that.

Reply:
Nice. Also I did some reading (http://vanseodesign.com/css/sass-the-import-directive/) and confirmed that you can miss off the extension and the preceding underscore and sass still finds the file OK (not that it explains the likely importing from a different location stuff)

Reply:
👍